src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/wwwroot/swagger/swagger-utils.js

```diff
@@ -14,17 +17,44 @@ const initLoginForm = () => {
 
     overrideSwaggerAuthorizeEvent(swagger);
     overrideSwaggerLogoutEvent(swagger);
+    tryAuthorizeWithLocalData(swagger);
     showLoginUI(swagger);
 }
 
+const tryAuthorizeWithLocalData = (swagger) => {
+    if (isAuthorized(swagger))
+        return;
+
+    const token = getCookie(AUTHORIZATION_TOKEN_KEY);
+    if (!token)
+        return;
+
+    const authorizationObject = getAuthorizationRequestObject(token);
+    swagger.authActions.authorize(authorizationObject);
+}
+
 const overrideSwaggerAuthorizeEvent = (swagger) => {
     const auth = swagger.authActions.authorize;
-    swagger.authActions.authorize = (args) => attachReloadPageToEvent(swagger, auth, args);
+    swagger.authActions.authorize = async (args) => {
+        const result = await auth(args);
+
+        if (!isCookieSet(AUTHORIZATION_TOKEN_KEY)) {
```

Comment:
the `isCookieSet` method is redundant, plz use the `getCookie` method directly here like this which has the same result:

```js
if (!getCookie(...)) {
...
{
```